### DISCUSSION TITLE
Integrating Sablier with NGINX on Kubernetes for Feature Branch Management

### DISCUSSION THREAD

**Discussion by mostafahussein** [sablierapp/sablier#409] [Show and tell]
Hello,

I recently published an article about integrating Sablier with NGINX on Kubernetes to manage feature branch deployments. You can read the full article here:
[Scaling on Demand: How Sablier Can Optimize Feature Branch Deployments for Frontend Applications](https://medium.com/@mostafahussein/scaling-on-demand-how-sablier-can-optimize-feature-branch-deployments-for-frontend-applications-c2cda3bdccd2).

Please note that the integration involves modifying the original sablier.js, which you can view here:
https://github.com/mostafahussein/nginx-sablier-njs.

Thanks

**Comment by acouvreur** [comment]
Thank you very much for your writing!
